SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/PlayerDisconnect/SpatialTestPlayerDisconnectReturnToMainMenu.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestPlayerDisconnectReturnToMainMenu.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "GameFramework/Character.h"
+#include "Kismet/GameplayStatics.h"
+#include "PlayerDisconnectController.h"
+#include "SpatialConstants.h"
+#include "SpatialFunctionalTestFlowController.h"
+
+ASpatialTestPlayerDisconnectReturnToMainMenu::ASpatialTestPlayerDisconnectReturnToMainMenu()
+{
+	Author = "Victoria Bloom";
+	Description = TEXT("Ensure players are cleaned up correctly when they disconnected by the return to main menu.");
+	TriggerEventWaitTime = 5.0f;
```

Comment:
Is this used?

Reply:
No, removed it and other unnecessary class variables.